cg/models/downsample/downsample_data.py

```diff
@@ -0,0 +1,161 @@
+"""Model for down sampling meta data."""
+import logging
+from pathlib import Path
+
+from cg.apps.downsample.utils import case_exists_in_statusdb
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import Priority
+from cg.store import Store
+from cg.store.models import ApplicationVersion, Family, Sample
+from cg.utils.calculations import multiply_by_million
+
+LOG = logging.getLogger(__name__)
+
+
+class DownsampleData:
+    def __init__(
+        self,
+        status_db: Store,
+        hk_api: HousekeeperAPI,
```

Comment:
It is a little counter intuitive to me that a class called `DownsampleData` require these parameters, it sounds like a pure data class. It works though, feel free to disregard :+1: Or maybe you think there is a better name for it

Reply:
I guess one could have one being the collector of data "CollectDownsampleData" and then the other being as Sebastian suggested a data class "DownsampleData"